locals.tf

```
@@ -1,45 +1,80 @@
-# Helm chart value overrides. They should be passed as a list of HCL structures
-# Important we don't use the default helm values it's blank instead locals used to override it with custom values.
 locals {
-  helm_values = [{
-    minio = {
-      mode = "standalone" # set the deployment mode of MinIO to standalone
-      resources = {
-        requests = {
-          memory = "128Mi" # specify the memory request for the container
+  self_signed_cert_enabled = var.cluster_issuer == "ca-issuer" || var.cluster_issuer == "letsencrypt-staging"
+
+  self_signed_cert = {
+    extraVolumeMounts = [
+      {
+        name      = "certificate"
+        mountPath = "/etc/ssl/certs/ca.crt"
+        subPath   = "ca.crt"
+      },
+    ]
+    extraVolumes = [
+      {
+        name = "certificate"
+        secret = {
+          secretName = "minio-tls"
         }
       }
-      # specify the ingress configuration for the MinIO console UI
-      consoleIngress = {
-        enabled = true
-        annotations = {
-          "cert-manager.io/cluster-issuer"                   = "${var.cluster_issuer}"
-          "traefik.ingress.kubernetes.io/router.entrypoints" = "websecure"
-          "traefik.ingress.kubernetes.io/router.middlewares" = "traefik-withclustername@kubernetescrd"
-          "traefik.ingress.kubernetes.io/router.tls"         = "true"
-          "ingress.kubernetes.io/ssl-redirect"               = "true"
-          "kubernetes.io/ingress.allow-http"                 = "false"
+    ]
+  }
+
+  oidc_config = merge(
+    {
+      oidc = {
+        enabled      = var.oidc.issuer_url != "" && var.oidc.client_id != "" && var.oidc.client_secret != ""
+        configUrl    = "${var.oidc.issuer_url}/.well-known/openid-configuration"
+        clientId     = var.oidc.client_id
+        clientSecret = var.oidc.client_secret
+        claimName    = "policy"
+        scopes       = "openid,profile,email"
+        redirectUri  = "https://minio.apps.${var.cluster_name}.${var.base_domain}/oauth_callback"
+        claimPrefix  = ""
+        comment      = ""
+      }
+  }, local.self_signed_cert_enabled ? local.self_signed_cert : null)
```

I know I'm being picky, but a correct indentation in my mind would be:
```suggestion
    },
    local.self_signed_cert_enabled ? local.self_signed_cert : null
  )
```
Now if `terraform fmt` thinks otherwise, you should ignore my comment and leave it as it suggests.